Update 8.1.0-1.md (#201)
percona/pxb-docs: docs/release-notes/8.1.0-1.md

## Improvement

> * [PXB-3155] : The Percona Server for MySQL 8.0 series implemented the keyring_vault plugin. In the Innovation series, the server shifts to a component infrastructure, the keyring's functionality remains the same. Percona XtraBackup 8.1.0-1 supports the component keyring_vault. Users are able to back up their encrypted data after they convert from the keyring_vault plugin to the keyring_vault component.

Reviewer: ```suggestion
* [PXB-3155] : The Percona Server for MySQL 8.0 series implemented the keyring_vault plugin. In the Innovation series, the server shifts to a component infrastructure, the keyring's functionality remains the same. Percona XtraBackup 8.1.0-1 supports the component keyring_vault. Users can back up their encrypted data after they convert from the keyring_vault plugin to the keyring_vault component.
```
